tests/compile-test.rs

```diff
@@ -23,3 +23,23 @@ fn run_mode(mode: &'static str) {
 fn compile_test() {
     run_mode("compile-fail");
 }
+
+#[test]
+fn dogfood() {
```

Comment:
The reason dogfood is a separate bash script is because it causes a lot of churn and deletes built deps. Not something you want to happen every time you run `cargo test`